examples/multimap/main.go

```diff
@@ -52,5 +52,11 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Printf("entry found:%t", ok)
+	fmt.Printf("entry found:%t\n", ok)
+	// Print number of values match given "key"
+	nOfValue, err := m.ValueCount(ctx, "key")
```

Comment:
Let's rename `nOfValue` to something better. How about `count`?

Reply:
Fixed, [8d5d17e](https://github.com/hazelcast/hazelcast-go-client/pull/734/commits/8d5d17e46c48d1f3b6dbecd61ba9881f11c6a06a).